components/handlers/add_module_handler.py

```diff
@@ -31,5 +31,9 @@ def POST(self):
         module_desc = data.description
         module_mc = data.mc
 
+        if not (check_string.check_code(module_code) and check_string.check_name(module_name)
```

Comment:
Consider adding the check_string methods to model.py, then call the methods from model instead (i.e. use model as a facade for check_string)